Sources/Event Logging/TracksService.h

```diff
@@ -11,6 +11,9 @@ extern NSString *const TrackServiceDidSendQueuedEventsNotification;
 @property (nonatomic, strong) TracksEventService *tracksEventService;
 @property (nonatomic, strong) TracksServiceRemote *remote;
 
+/*! Used to configure the ExPlat `platform` property. If this property is `nil`, the `eventNamePrefix` property is used instead. Default is `nil`. */
+@property (nonatomic, copy, nullable) NSString *platform;
```

Comment:
The `WordPress-iOS` repo will set this to `wpios` for both Jetpack and WordPress.